### ISSUE TITLE
ocsng volumes unsorted columns

### ISSUE THREAD

**Issue by pchamo** [glpi-project/glpi#5311]
I have GLPI 9.3.3 and glpi-ocsinventoryng-1.5.6 under CentOS 7 working. Syncronization is working. I have 5 windows server with identical hardware configuration, 2 disk each one, the first one has 100 GB for OS and the second one has 120 GB. Querying the servers after adding some volumes columns to the default view, some data are shown unsorted, please see:

http://prntscr.com/mevddh

I modified the file glpi/inc/search.class.php as below, and it seems to me that it solves the bug:

```
[root@ocs inc]# diff search.class.php search.class.php.ori
2858c2858
<                                                         `$table$addtable`.`id`) ORDER BY $ `$table$addtable`.`id`
---
>                                                         `$table$addtable`.`id`)
2883c2883
<                                                '".self::SHORTSEP."',$tocomputeid) ORDER BY $tocomputeid SEPARATOR '".self::LONGSEP."')
---
>                                                '".self::SHORTSEP."',$tocomputeid) SEPARATOR '".self::LONGSEP."')
```
And here the result:

http://prntscr.com/mevglb

Could you please include the proposed fix in GLPI? Thank you.

**Comment by trasher** [comment]
Hi,

Please open a pull request wit your proposal, so tests can be run. Thank you

**Comment by cedric-anne** [comment]
Hi,

Can you test #5529 ?
Which database server are you using ? And which version ?

Regards

**Comment by cedric-anne** [comment]
Fixed by 9335774b0e19d2d4c0cdef3a4aa2b8c6dd55dedf .
